src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -42,6 +42,21 @@ public class LanguageServer
 
         private TaskCompletionSource<bool> serverCompletedTask;
 
+        private static string checkHelpScript = @"
+[CmdletBinding()]
+param (
+    [String]$CommandName
+)
+Try {
+    Microsoft.PowerShell.Core\Get-Command $CommandName -ErrorAction Stop | Out-Null
+} catch [System.Management.Automation.CommandNotFoundException] {
+    $PSCmdlet.ThrowTerminatingError($PSItem)
+}
+try {
+    Microsoft.PowerShell.Core\Get-Help $CommandName -Online | Out-Null
```

Comment:
echo: Out-Null

Reply:
Yeah, I debated Out-Null or just assigning to $null. I think I recall reading somewhere that assignment is faster (but not by much, especially for this type of command)

Reply:
Casting to `[void]` or assigning to `$null` is faster in Windows PowerShell, but I recall reading that we now recoginise the pipe-to-`Out-Null` case in PS Core and have optimised it out too. Can't find a source on that though.

Reply:
https://stackoverflow.com/questions/5260125/whats-the-better-cleaner-way-to-ignore-output-in-powershell is one source

Reply:
Yeah it's faster at scale, but with a single item it doesn't matter.  I typically default to `$null` assignment but I've been trying to break the habit since `Out-Null` is significantly easier to explain to beginners.

(I'm not suggesting you change it back, I'd rather `$null = ` than the fully qualified name every time 😄)